core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -87,4 +117,8 @@ public void validateReferencedColumns(Schema schema) {
   public MetricsMode columnMode(String columnAlias) {
     return columnModes.getOrDefault(columnAlias, defaultMode);
   }
+
+  public boolean allDefault() {
```

Comment:
Not sure where this is used?

Reply:
Good catch, I don't remember why I added this myself it's been so long. But yeah I don't see any callers so I'll take it out.